gcloud/bigtable/client.py

```diff
@@ -168,7 +166,8 @@ def copy(self):
         :rtype: :class:`.Client`
         :returns: A copy of the current client.
         """
-        copied_creds = copy.deepcopy(self._credentials)
+        credentials = self._credentials
+        copied_creds = credentials.create_scoped(credentials.scopes)
```

Comment:
:+1: I'd really like to clean up the way `ServiceAccountCredentials` uses / copies its non-private info.